assets/javascripts/discourse/initializers/hook-decrypt-post.js.es6

```diff
@@ -371,106 +371,121 @@ export default {
       api.reopenWidget("post-contents", {
         html(attrs, state) {
           const topicId = attrs.topicId;
-          if (attrs.id === -1 || !hasTopicTitle(topicId)) {
-            return this._super(...arguments);
+          if (attrs.id !== -1 && hasTopicTitle(topicId)) {
+            decryptPost.bind(this)(attrs, state, topicId);
```

Comment:
Nitpick: I think you could have one less function call here, by using `call` instead of `bind`.

```suggestion
            decryptPost.call(this, attrs, state, topicId);
```